RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -217,11 +237,21 @@ private string GeneratePluginForPackage(string outputDir, string language, strin
                 createdJarFilePath = BuildPlugin(definition, outputDir);
             }
 
+            LogMessageForGeneratedRules(generatedRulesTemplateFile);
             LogMessageForGeneratedSqale(generatedSqaleFile);
 
             return createdJarFilePath;
         }
 
+        private void LogMessageForGeneratedRules(string generatedFile)
+        {
+            if (generatedFile != null)
```

Comment:
Isn't it better to also log (another message) when the file is null?

Reply:
In that case the user will have supplied the file name as a parameter, and a message will have been logged in line 218 above.